internal/handlers/pg/msg_find.go

```diff
@@ -154,3 +169,67 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 
 	return &reply, nil
 }
+
+// getMaxTimeMSParameter gets maxTimeMS optional parameter.
+func getMaxTimeMSParameter(document *types.Document) (int32, error) {
+	var maxTimeMS int32
+	maxTimeMSParam, err := common.GetOptionalParam(document, "maxTimeMS", maxTimeMS)
+	if err != nil {
+		var commonErr *common.Error
+		if errors.As(err, &commonErr) {
+			return 0, getErrorForInvalidMaxTimeMS(document)
+		}
+
+		return 0, err
```

Comment:
We disard the error we get from `common.GetOptionalParam`, so this could be simplified to:
```suggestion
		return 0, getErrorForInvalidMaxTimeMS(document)
```

Reply:
Fixed